File: Basics/BasicMaths/CountDigits/count.js
const countDigits = (n) => {
  let x = n;
  let count = 0;
  while (x) {
    x = Math.floor(x / 10);
    count++;
  }
  return count;
};

let num = 12345;
console.log(`Total digits in ${num} is ${countDigits(num)}`);

File: Basics/BasicMaths/CountDigits/count.cpp
#include<iostream>
using namespace std;

int countDigits(int n){
    int x=n;
    int count = 0;
    while(x){
       x=x/10;
       count++;
    }
    return count;
}
int main(){
    int n=12345;
    cout<<"Total digits in "<< n <<" is "<<countDigits(n);
    return 0;
}